hash/mac_flop_orig.xml

```diff
@@ -2272,4 +2272,1114 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="archon">
+		<description>Archon</description>
+		<year>1986</year>
+		<publisher>Electronic Arts</publisher>
+		<info name="programmer" value="Anne Westfall, Jon Freeman, Paul Reiche III, and Jim Nitchals" />
+		<info name="usage" value="Self-boots and runs on Mac 512K, Mac 512Ke, and Mac Plus." />
+		<sharedfeat name="compatibility" value="mac512k,mac512ke,macplus" />
+		<!-- Dump released: 2024-11-09 -->
+		<!-- strategy game -->
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="649022">
+				<rom name="archon.moof" size="649022" crc="1613732e" sha1="58e128f28ebb7935de83fcf3c2fead8305cb1944" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="fractact">
+		<description>Fraction Action</description>
+		<year>1985</year>
+		<publisher>Unicorn Software</publisher>
+		<info name="programmer" value="David Howard Flatman and Ray Tripamer" />
+		<info name="usage" value="Self-boots and runs on Mac 128K, Mac 512K, Mac 512Ke, and Mac Plus." />
+		<sharedfeat name="compatibility" value="mac128k,mac512k,mac512ke,macplus" />
+		<!-- Dump released: 2025-01-14 -->
+		<!-- educational game -->
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="692196">
+				<rom name="fraction action.moof" size="692196" crc="7a32ac3e" sha1="b102d78731228bcbf165d6a28c7984b215e165c8" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="bitstyl">
+		<description>The Bitstream Typeface Library (version 2.0)</description>
+		<year>1991</year>
+		<publisher>Bitstream</publisher>
+		<info name="developer" value="Bitstream" />
+		<info name="usage" value="Each font is available in AFM, Suitcase, and Type 1 format." />
+		<info name="version" value="2.0" />
+		<sharedfeat name="compatibility" value="mac128k,mac512k,mac512ke,macplus,macse,macsefd,macclasc" />
+		<!-- Dump released: 2025-03-28 -->
+		<!-- font collection -->
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="AFMS Disk 1 - aachen - americana" />
+			<dataarea name="flop" size="1312544">
+				<rom name="the bitstream typeface library - afms disk 1 aachen - americana.moof" size="1312544" crc="88890226" sha1="1fd16f1a30900c42cac2959e5cfd954274dc3c11" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="AFMS Disk 2 - amerigo, bitstream - aurora" />
+			<dataarea name="flop" size="1312553">
+				<rom name="the bitstream typeface library - afms disk 2 amerigo, bitstream - aurora.moof" size="1312553" crc="8a1a0432" sha1="075d9fea98e331dfcc24bd11db8b8d212d4a32c4" />
+			</dataarea>
+		</part>
```

Comment:
Bitstream typeface collections usually used title case on the disk labels, e.g. see some from a pack-in TrueType collection here: https://www.alamy.com/bitstream-font-pack-true-type-25-for-microsoft-windows-31-in-set-of-four-disks-from-the-90s-image460076924.html

Can we get title case in the part IDs here?